client/src/locales/es/messages.po

```diff
@@ -51,7 +51,7 @@ msgstr "(esto puede tardar un rato)"
 msgid "({browserType, select, mobile {tap} other {click}} to view details)"
 msgstr "({browserType, select, mobile {pulsa} other {haz clic}} para ver detalles)"
 
-#: src/containers/HomePage.tsx:123
```

Comment:
are these line number references automatically changed when `.tsx` files are modified?

Reply:
yeah, you just run `yarn lingui extract` and these files are regenerated. Some of the details of exactly how it keeps track of everything and what the `#~` type things signify I'm not actually sure about.